addons/event/views/event_views.xml

```diff
@@ -232,8 +231,7 @@
                                     <tree string="Communication" editable="bottom">
                                         <field name="sequence" widget="handle"/>
                                         <field name="notification_type"/>
-                                        <field name="template_model_id" invisible="1"/>
```

Comment:
for safety, it's better to leave the field in the template (even if it doesn't do anything and is invisible), because:

- removing it could break a customization that did an XPATH using that field
- some other customization could rely on the field being in the view

Reply:
in this case looks good :+1: , I thought the access right error was cause by the READ call to ir.model (and that this did not cause an error because the field has the attribute "compute_sudo"